src/modules/gamewisp/global-emotes.js

```diff
@@ -0,0 +1,65 @@
+const $ = require('jquery');
+const settings = require('../../settings');
+
+const AbstractEmotes = require('../emotes/abstract-emotes');
+const Emote = require('../emotes/emote');
+
+const provider = {
+    id: 'gw-global',
+    displayName: 'GameWisp Global Emotes',
+    badge: 'https://d32y8axfzs6sv8.cloudfront.net/static/gamewisp_transparent_18px.png'
+};
+
+class GWGlobalEmotes extends AbstractEmotes {
+    constructor() {
+        super();
+
+        settings.add({
+            id: 'gwEmotes',
+            name: 'GameWisp Emotes',
+            defaultValue: false,
+            description: 'Use GameWisp emotes'
+        });
+
+        if (settings.get('gwEmotes')) {
```

Comment:
move this check to within the update function instead of including it here and below.

also I kinda prefer an explicit `true` value check